tests/unit/views/notifications/test_organization_admin_notification.py

```diff
@@ -0,0 +1,231 @@
+"""Test template strings in project crew membership notifications."""
+# pylint: disable=too-many-arguments
+
+from pytest_bdd import given, parsers, scenarios, then, when
+
+from funnel import models
+from funnel.models.membership_mixin import MEMBERSHIP_RECORD_TYPE
+
+scenarios('organization_admin_notification.feature')
+
+
+@given(
+    "Vetinari is an owner of the Ankh-Morpork organization",
+    target_fixture='vetinari_admin',
+)
+def given_vetinari_owner_org(user_vetinari, org_ankhmorpork) -> None:
+    assert 'owner' in org_ankhmorpork.roles_for(user_vetinari)
+    vetinari_admin = org_ankhmorpork.active_owner_memberships[0]
+    assert vetinari_admin.user == user_vetinari
+    return vetinari_admin
```

Comment:
Since this fixture is a dupe of the one in the other file, it can be moved to a file named `conftest.py` in the same folder (not the main `conftest.py` but local to this folder). I think pytest-bdd will process that.